content/blog/announcing-pulumi-esc-github-action/index.md

```diff
@@ -0,0 +1,92 @@
+---
+title: "Announcing the Pulumi ESC GitHub Action"
+date: "2025-03-05"
+meta_desc: "Simplify Secrets Management and CI/CD with Pulumi ESC."
+authors: 
+  - "komal-ali"
+tags:
+  - esc
+  - secrets
+  - github
+  - features
+---
+
+We’re excited to share our latest addition to the Pulumi Ecosystem: the [Pulumi ESC GitHub Action](https://github.com/marketplace/actions/esc-action). This new GitHub Action streamlines how you manage secrets, configurations, and CI/CD tasks using [Pulumi ESC](/product/esc), helping your team securely scale and automate your software delivery pipelines.
+
+---more
+
+## Why We Built It
+
+Secrets management continues to be a challenge for organizations of all sizes. Storing long-lived credentials in multiple places (e.g., GitHub, third-party CI systems, and internal servers) often leads to “secret sprawl” and potential security gaps. With Pulumi ESC, you can centralize and manage short-lived dynamic credentials, rotate them automatically, and ensure secrets don’t persist beyond their necessary lifetime.
```

Comment:
```suggestion
Secrets management continues to be a challenge for organizations of all sizes. Storing long-lived credentials in multiple places (e.g., GitHub, third-party CI systems, and internal servers) often leads to “secret sprawl” and potential security gaps. With Pulumi ESC, you can migrate to dynamic credentials, and ensure secrets don’t persist beyond their necessary lifetime.  If your process requires longer-lived static credentials, with ESC you can manage credentials in a central place and rotate them automatically.
```

We should have a link to secrets rotation blog post or docs.

Reply:
It's linked later in the doc, but I can link again here.